website/docs/BasicInterfaces/Handler.md

```diff
@@ -146,7 +145,7 @@ NOTE: when using this method, you must call [`agent.close()`](/docs/basic-interf
 
 - options `object`. Accepts any of the following:
   - name `string`. This is used to generate a unique sessionName.
-  - browserEmulatorId `string` defaults to `chrome-latest`. Emulates a specific browser engine version.
+  - browserEmulatorId `string` defaults to `default-browser-emulator`. Emulates the properties that help SecretAgent look like a normal browser.
   - humanEmulatorId `string`. Drives human-like mouse/keyboard movements.
```

Comment:
Should we remove mention of this while it's in limbo?

Reply:
remove mention of humanEmulatorId?

Reply:
Wondering if we should. It seems unclear what to do with this parameter in the current setup.

Reply:
Like, is it now a humanEmulator"PluginId"?

Reply:
How is it any different than before? Or are you suggesting we go ahead and
remove HumanEmulator as a concept?

Reply:
I'm not sure I have a good thought on fully removing. I do think there are times you'd want to direct a given scrape to use a specific human profile. It seems like a difference from before is we had a page explaining the 3 built-in human emulators you could switch between, and you installed additional ones into "HumanEmulators.use". Now, you have to install a plugin, and then refer to the pluginId? So you're not really referring to a humanEmulatorId, but a pluginId, right?

Reply:
A humanEmulatorId is a pluginId as in a HumanEmulator extends CorePlugin.

Reply:
So maybe the docs should at least be updated to say "The pluginId that should be used to drive human-like mouse/keyboards movements for the session". And did you say there was a way to disable the default plugin? Like passing null or something?

Reply:
Fixed.